### PR TITLE
MVP guidelines for Numerical Input

### PR DESCRIPTION
**Pull request by kpoulincoveo**
### Proposed Changes

revised all guidelines

### Potential Breaking Changes

N\A

### Acceptance Criteria
This I don't know 👇 
-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

**Review by aiheon — APPROVED** [review]

**Comment by kpoulincoveo** [comment]
@jlevesquecoveo to be revised (not urgent)

**Review by gdostie — APPROVED** [review]

File: packages/website/docs/NumericInput.md [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
+
+If the input is set within a range, consider indicating the minimum and maximum values, or only one of them, in a help text, e.g. "Maximum of 20".
 
-[Text Input](#/components/Input)
+## Feedback and Validation
 
-[Radio Buttons](#/components/Radio)
+Numerical inputs are best when the value doesn't require decimal or fraction. If decimals are expected to be frequently set by users, make sure to show **the decimal point** in the default value, e.g.: "20.0", to indicate which format is accepted.
+
+Always shows in the validation message both the minimal and maximal limit if the user tries to enter a number outside the acceptable range.
```
Comment (jlevesquecoveo) [inline packages/website/docs/NumericInput.md:23]:
```suggestion
Display a validation message when users enter a value outside of the expected range. This message should include the maximal and minimal values accepted.
```

File: packages/website/docs/NumericInput.md [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+
+Determine the stepper increment relative to the range. For instance, if the range is 1-10, the increments should be of 1. If the range is 1-100, the increments should be of 10.
```
Comment (jlevesquecoveo) [inline packages/website/docs/NumericInput.md:25]:
```suggestion
Determine the stepper increment relative to the range. For instance, if the range is 1-10, the increments should be of 1. If the range is 1-100, the increments should be of 10.
```
@kpoulincoveo Not sure I understand the advice here. Can we be more specific? Should we recommend increments of 10%?

File: packages/website/docs/NumericInput.md [hunk 3]
```diff
@@ -1,17 +1,34 @@
...
+
+
+---
+
+## Related Components
 
-[Dropdown List](#/components/Drop)
+If your use case doesn't match the guidelines above, consider using one of the following components instead:
```
Comment (jlevesquecoveo) [inline packages/website/docs/NumericInput.md:32]:
```suggestion
If value precision is unimportant, consider using a [slider](#/form/Sliders) instead.
```

File: packages/website/docs/NumericInput.md [hunk 4]
```diff
@@ -1,17 +1,34 @@
...
 
-[Toggle](#/components/FlatSelect)
+-   [Sliders](#/form/Sliders) when the precision of the value is not important.
```
Comment (jlevesquecoveo) [inline packages/website/docs/NumericInput.md:34]: Delete if you accept the above suggestion

Comment (aiheon) [reply packages/website/docs/NumericInput.md:34]: @kpoulincoveo

File: packages/website/docs/NumericInput.md (hunk 3 above)
Comment (aiheon) [reply packages/website/docs/NumericInput.md:32]: @kpoulincoveo

File: packages/website/docs/NumericInput.md (hunk 2 above)
Comment (kpoulincoveo) [reply packages/website/docs/NumericInput.md:25]: I think we can suggest an increment of 10% but am not sure it always gonna be the case.

File: packages/website/docs/NumericInput.md (hunk 4 above)
Comment (kpoulincoveo) [reply packages/website/docs/NumericInput.md]: It was updated

File: packages/website/docs/NumericInput.md [hunk 5]
```diff
@@ -1,17 +1,33 @@
...
+If the value to select must be within a range, consider indicating the minimum and maximum values, or only one of them, in a help text (e.g., "Maximum of 20").
 
-[Text Input](#/components/Input)
+# Feedback and Validation
+
+Ideally, the value to enter in a numerical input should be an integer. If you expect users to frequently enter decimals, **show the decimal point** in the default value (e.g.:, "20.0") to indicate the accepted format.
+
+Display a validation message when users enter a value outside of the expected range. This message should include the maximal and minimal values accepted.
+
+Determine the stepper increment relative to the range. An increment of 10% is generally acceptable. For instance, if the range is 1-10, the increments should be of 1. If the range is 1-5000, the increments of 500 would be acceptable.
```
Comment (jlevesquecoveo) [inline packages/website/docs/NumericInput.md:25]:
```suggestion
Determine the stepper increment relative to the range. An increment of 10% is generally acceptable. For instance, if the range is 1-10, the increments should be of 1. If the range is 1-5000, increments of 500 would be acceptable.
```
@kpoulincoveo This is my last suggestion!

**Review by jlevesquecoveo — APPROVED** [review]

**Comment by kpoulincoveo** [comment]
@gdostie  @aiheon  Ready to merge

**Pull request merged**